services/huaweicloud/src/main/java/org/eclipse/osc/orchestrator/plugin/huaweicloud/builders/HuaweiImageBuilder.java

```diff
@@ -67,4 +80,93 @@ public boolean destroy(BuilderContext ctx) {
         // TODO: destroy the temporary images here or after resources created.
         return true;
     }
+
+    protected TFExecutor prepareExecutor(BuilderContext ctx) {
+        if (ctx == null) {
+            log.error("BuilderContext is invalid.");
+            throw new BuilderException(this, "Builder context is null.");
+        }
+        Map<String, String> envCtx = ctx.get(new HuaweiEnvBuilder(ocl).name());
+        if (envCtx == null) {
+            log.error("Dependent builder: {} must build first.", new HuaweiEnvBuilder(ocl).name());
+            throw new BuilderException(this, "HuaweiEnvBuilder context is null.");
+        }
+        TFExecutor executor = new TFExecutor(envCtx);
+        executor.createWorkspace(name());
+        executor.createTFScript(""
+            + "resource \"huaweicloud_vpc\" \"vpc\" {\n"
```

Comment:
How about using %n instead of \n

Reply:
I tried `%n` before this PR, but it seems not to work well. 
How about leaving it as it is? I will check it sometime.

Reply:
Sure